README.md

```diff
@@ -92,11 +92,13 @@ For projects hosted by [GitLab](https://gitlab.com) reference [Microsoft.SourceL
 
 ### Bitbucket.org
 
-For projects hosted on [Bitbucket.org](https://bitbucket.org) in git repositories reference [Microsoft.SourceLink.Bitbucket.Git](https://www.nuget.org/packages/Microsoft.SourceLink.Bitbucket.Git) package: 
+For projects hosted on [Bitbucket.org](https://bitbucket.org) in git repositories reference [Microsoft.SourceLink.Bitbucket.Git](https://www.nuget.org/packages/Microsoft.SourceLink.Bitbucket.Git) package
+and add Bitbucket host configuration. Including flag whether it is Enterprise Edtition or Cloud. For Enterprise Edition provide its version and change value in include attribute to your Bitbucket server name: 
 
 ```xml
 <ItemGroup>
   <PackageReference Include="Microsoft.SourceLink.Bitbucket.Git" Version="1.0.0-beta2-18618-05" PrivateAssets="All"/>
+  <SourceLinkBitbucketGitHost Include="bitbucket.org" EnterpriseEdition="true" Version="4.1" />
```

Comment:
><SourceLinkBitbucketGitHost Include="bitbucket.org" EnterpriseEdition="true" Version="4.1" /> [](start = 2, length = 93)

This should be default. Customer who uses repository hosted on bitbucket.org shouldn't need to specify anything in their project file.

Reply:
When using a server hosted in different domain we should default to enterprise edition and the latest version with the option to specify an older version.

---
In reply to: [280532565](https://github.com/dotnet/sourcelink/pull/246#discussion_r280532565) [](ancestors = 280532565)

Reply:
The default host is specified in `src\SourceLink.Bitbucket.Git\build\Microsoft.SourceLink.Bitbucket.Git.props`, so I think you just need to update it with the attributes.

---
In reply to: [280533129](https://github.com/dotnet/sourcelink/pull/246#discussion_r280533129) [](ancestors = 280533129,280532565)

Reply:
Done. 
Now for bitbucket.org cloud hosting there is no need to provide SourceLinkBitbucketGitHost configuration in csproj. 
When the domain is different than bitbucket.org it defaults to enterprise edition and the latest version.